notebooks/408-cocosnet-model/408-cocosnet-ai-painting.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    demo.launch(share=True)
</pre><p>Is it possibly to use local instance if it Is applicable like in https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/237-segment-anything</p><p>At least I think it is better to remove this cell output (now any person can run demo on your machine using provided link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>